homeassistant/components/light/hue.py

```diff
@@ -340,8 +361,41 @@ def turn_off(self, **kwargs):
         elif self.bridge_type == 'hue':
             command['alert'] = 'none'
 
-        self.bridge.set_light(self.light_id, command)
+        self._command_func(self.light_id, command)
 
     def update(self):
         """Synchronize state with bridge."""
         self.update_lights(no_throttle=True)
+
+
+class HueLightGroup(HueLight):
+    """Representation of a Hue light group."""
+
+    def __init__(self, light_id, info, bridge, update_lights,
+                 bridge_type):
+        """Initialize the light group."""
+        super().__init__(light_id, info, bridge, update_lights,
+                         bridge_type, False)
+        self._command_func = self.bridge.set_group
+
+    @property
+    def brightness(self):
+        """Return the brightness of this light group between 0..255."""
+        return self.info['action'].get('bri')
+
+    @property
+    def xy_color(self):
+        """Return the XY color value."""
+        return self.info['action'].get('xy')
+
+    @property
+    def color_temp(self):
+        """Return the CT color value."""
+        return self.info['action'].get('ct')
+
+    @property
+    def is_on(self):
+        """Return true if any light of the group is on."""
+        self.update_lights()
```

Comment:
Yes, it has to go. I guess it was a no-op because of throttling but yes, we should not do any I/O inside properties because it will block anything from happening in HASS (including even setting a state!).